website/util/sanitize.py

```diff
@@ -58,3 +58,17 @@ def _ensure_clean(value):
             raise ValueError
 
     return escape_html(data)
+
+
+def unescape_html(s):
+    """
+    Return a string without html escape characters.
+
+    :param s: A string
+    :return: A string without html escape characters
+
+    """
+    s = s.replace('&amp;', '&')
```

Comment:
It would be cleaner to define a mapping between escape sequences and their corresponding characters.

```python
safe_characters = {
    '&amp;': '&'
    # ...
}
for escape_sequence, character in safe_characters.items():
    s = s.replace(escape_sequence, character)
```

This makes it easier to add additional characters in the future.